.github/CODEOWNERS

```diff
@@ -49,6 +49,7 @@ yarn.lock                                           @backstage/reviewers @backst
 /plugins/kubernetes                                 @backstage/reviewers @backstage/warpspeed
 /plugins/kubernetes-*                               @backstage/reviewers @backstage/warpspeed
 /plugins/newrelic-dashboard                         @backstage/reviewers @mufaddal7
+/plugins/org-react                                  @backstage/reviewers
```

Comment:
@Rugvip Is this the right owner? Does it need an owner at all?

Reply:
You can skip the explicit owner, there's a catch-all at the bottom that handles this